src/appMain/sdl_preloaded_pt.json

```diff
@@ -2291,6 +2387,15 @@
                 "priority": "NONE",
                 "default_hmi": "NONE",
                 "groups": ["BaseBeforeDataConsent"]
+            },
+            "pre_consent_passengersRC": {

```

Comment:
In that case, a "pre_consent_RC" is more appropriate than "pre_consent_passengersRC"